cmd/root.go

```diff
@@ -30,11 +31,12 @@ const (
 //nolint:gocognit,funlen // This function is the CLI entrypoint and it's expected to be long.
 func NewRootCommand() *cobra.Command {
 	var (
-		level        logconfig.Level // log level.
-		outputScan   bool            // print result of scan as JSON to stdout.
-		skippedNs    []string        // list of namespaces to be skipped from scan.
-		insecureSSL  bool            // skip SSL cert validation when connecting to PolicyServers endpoints.
-		disableStore bool            // disable storing the results in the k8s cluster.
+		level        string       // log level.
+		outputScan   bool         // print result of scan as JSON to stdout.
+		skippedNs    []string     // list of namespaces to be skipped from scan.
+		insecureSSL  bool         // skip SSL cert validation when connecting to PolicyServers endpoints.
+		disableStore bool         // disable storing the results in the k8s cluster.
+		slogger      *slog.Logger // slog based logger to be used
```

Comment:
there is no need to declare this variable here

Reply:
Ack, done.